easybuild/easyconfigs/g/GLIMPSE/GLIMPSE-2.0.0-GCC-12.2.0.eb

```diff
@@ -26,19 +26,15 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('Boost', '1.81.0'),
+    ('Boost', '1.82.0'),
     ('bzip2', '1.0.8'),
     ('cURL', '7.86.0'),
-    ('HTSlib', '1.17'),
+    ('HTSlib', '1.15.1'),
     ('XZ', '5.2.7'),
     ('zlib', '1.2.12'),
 ]
 
-prebuildopts = 'ls */makefile | xargs sed -i "s/^CXXFLAG=.*/CXXFLAG=${CXXFLAGS}/g" && '
-prebuildopts += 'ls */makefile | xargs sed -i "s/^LDFLAG=.*/LDFLAG=${CXXFLAGS}/g" && '
-buildopts = 'CXX="$CXX" HTSSRC=${EBROOTHTSLIB} HTSLIB_LIB="-lhts -lcurl" BOOST_INC=$EBROOTBOOST/include '
-buildopts += 'BOOST_LIB_IO="-lboost_iostreams" BOOST_LIB_PO="-lboost_program_options" '
-buildopts += 'BOOST_LIB_SE="-lboost_serialization" '
+prebuildopts = 'export COMPILATION_ENV=unil-dcsr && export XZ_ROOT=$EBROOTXZ && export BZIP2_ROOT=$EBROOTBZIP2 && export CURL_ROOT=$EBROOTCURL && export HTSLIB_ROOT=$EBROOTHTSLIB && export HTSSRC=$EBROOTHTSLIB && export BOOST_LIB_SE=$BOOST_ROOT/lib/libboost_serialization.a && export BOOST_LIB_PO=$BOOST_ROOT/lib/libboost_program_options.a && export BOOST_LIB_IO=$BOOST_ROOT/lib/libboost_iostreams.a && export BOOST_INC=$BOOST_ROOT/include/ && '
```

Comment:
Would be interested in your reasoning here too - the result seems a bit less readable than what we had before.